aip/auth/4114.md

```diff
@@ -100,6 +100,19 @@ the form of an X.509 cert followed immediately by the private key:
   ...
   -----END PRIVATE KEY-----
 
+### Obtaining the Default Device Certificate via ECP
+
+The Enterprise Certificate Proxy (ECP) project is the newly recommended way to
+procure device certificates. It has two major advantages compared to the legacy
+SecureConnect mechanism:
+
+1.  Allows usage of enterprise certs and private keys stored in native keystores and TPMs
+    instead of relying on self-signed certs.
+1.  Delegates signing operations to keystores, so private keys never leave the security realm.
+
+Please see [ECP Public Documentation][2] for details on ECP configuration. ECP is
+fully supported in Google Golang libraries and gCloud SDK as of 2023.
```

Comment:
Are we sure we want to mention which languages/surfaces support it here and not in say the doc site where there are already examples and better means of expressing that? Otherwise we will need to keep this updated whenever a new language/surface supports it.

That said, I'm not opposed to including it if no where else is reasonable - it's good to know and not have to ask around.

Reply:
This is a good point - I will omit the last part since the AIP is a recommendation and not a status tracker. Ideally there should be a separate public documentation today on which feature is supported in various clients by language, but no such thing exists AFAIK.

Reply:
Yep I agree and TBH the client library AIPs don't have this level of tracking though we regularly ask each other which language supports which feature.